java/arcs/core/data/testdata/Manifest2ProtoTest.textproto

```diff
@@ -0,0 +1,303 @@
+# This files is a part of 3 files suite of testdata:
+# - Manifest2ProtoTest.arcs
+# - Manifest2ProtoTest.pb.bin
+# - Manifest2ProtoTest.textproto (this file)
+#
+# On the TypeScript side we serialize .arcs file and validate it equals the .pb.bin file.
+# On the Kotlin side we deserialize .pb.bin and validate it equals parsed .textproto file.
+# This provides an additional guarantee beyond unit tests that at least all the constructs used
+# in the .arcs file can be serialized in TS and deserialized in Kotlin to the extent that they
+# are present in the .textproto file.
```

Comment:
Probably not worth repeating this text in so many places. You could just say something like `# Expected output from Manifest2ProtoTest.arcs`, but up to you.

Reply:
I think it's useful enough to keep.